modules/home/assets/js/components/home-screen.js

```diff
@@ -13,9 +14,10 @@ const HomeScreen = ( props ) => {
 					topData={ props.homeScreenData.top }
 					createNewPageUrl={ props.homeScreenData.create_new_page_url }
 				/>
-				<Container disableGutters={ true } sx={ { display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, justifyContent: 'space-between' } }>
-					{ /* Placeholder container to ensure layout until other components are added */ }
-					<Container maxWidth="md"></Container>
+				<Container disableGutters={ true } sx={ { display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, justifyContent: 'space-between', gap: 3 } }>
+					<Container maxWidth="md" disableGutters={ true }>
```

Comment:
``
```suggestion
					<Container disableGutters={ true }>
```

Reply:
maxWidth = md removed, lg we need to control max width of the container of 1200px as for design requirements

Reply:
I think there are other `flex `ways to handle it.
Can we give the sidebar a fixed `width` of `305px`?